src/main/create-cluster/allowed-resources.injectable.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable } from "@ogre-tools/injectable";
+import { computed } from "mobx";
+import { allowedResourcesInjectionToken } from "../../common/cluster-store/allowed-resources-injection-token";
+
+// TODO: Figure out implementation for this later.
```

Comment:
Important bit here: Dummy implementation for main till we have real one.

Reply:
I don't think we can ever have a real one. What I think is that we should try and move `isAllowedResource` to be only within `renderer`.

Reply:
As we discussed in huddle, we surely can. When we have removed the iframe, we can call e.g. `navigateToPods(clusterId)` which then will have the requirement of checking if cluster has pods. Or even application menu containing all clusters and ways to navigate to pages in them. But that's enough speculation :) We'll get back on this when the time comes.